scalanativelib/test/src/HelloNativeWorldTests.scala

```diff
@@ -169,31 +175,35 @@ object HelloNativeWorldTests extends TestSuite {
         )
       }
 
-      'utest_21112_038_debug - (checkUtest("2.11.12", "0.3.8", ReleaseMode.Debug))
-      'utest_21112_038_release - (checkUtest("2.11.12", "0.3.8", ReleaseMode.Release))
-      'scalaTest_21112_038_debug - (checkScalaTest("2.11.12", "0.3.8", ReleaseMode.Debug))
-      'scalaTest_21112_038_release - (checkScalaTest("2.11.12", "0.3.8", ReleaseMode.Release))
+      'utest_21112_039_debug - (checkUtest(scala211, scalaNative03, ReleaseMode.Debug))
+      'utest_21112_039_release - (checkUtest(scala211, scalaNative03, ReleaseMode.Release))
+//      No utest artifact for Scala Native 0.4.0-M2 published yet
+//      'utest_21112_040M2_debug - (checkUtest(scala211, scalaNative04, ReleaseMode.Debug))
+//      'utest_21112_040M2_release - (checkUtest(scala211, scalaNative04, ReleaseMode.Release))
+
+//      Scalatest dropped Scala Native 0.3 support
+      'scalaTest_21112_040M2_debug - (checkScalaTest(scala211, scalaNative04, ReleaseMode.Debug))
+//      Disabled since it consumes too much memory      
+//      'scalaTest_21112_040M2_release - (checkScalaTest(scala211, scalaNative04, ReleaseMode.Release))
     }
 
     def checkRun(scalaVersion: String, scalaNativeVersion: String, mode: ReleaseMode): Unit = {
-      val task = HelloNativeWorld.helloNativeWorld(scalaVersion, scalaNativeVersion, mode).run()
-
+      val task = HelloNativeWorld.helloNativeWorld(scalaVersion, scalaNativeVersion, mode).nativeLink
       val Right((_, evalCount)) = helloWorldEvaluator(task)
 
       val paths = Evaluator.resolveDestPaths(
         helloWorldEvaluator.outPath,
         task.ctx.segments
       )
-      val log = os.read(paths.log)
-      assert(
-        evalCount > 0,
-        log.contains("Scala Native")
-      )
+      val stdout = ammonite.ops.%%("./out")(paths.out / 'dest).out.lines
```

Comment:
I would be great if you could use `os.proc` instead of `ammonite.ops` here.

Reply:
✅Done

Reply:
Added again the test on `evalCount > 0` I had removed by mistake.